bibliopixel/util/duration.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+"""Convert a string duration to a number of seconds.
+"""
+
+SI_PREFIXES = (
+    (10 ** 24, 'yotta', 'Y'),
+    (10 ** 21, 'zetta', 'Z'),
```

Comment:
Well, I just pasted an existing table into here with a little editing.  :-D

Cost me nothing to do all of them, so why not?   I personally want yottasecond animations, and I run them very very fast, yo.

Easiest to do it all.